conda/basin_setup/basin_setup_quick.yaml

```diff
@@ -0,0 +1,18 @@
+name: basin_setup
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - colorama
+  - coloredlogs
+  - geopandas
+  - netcdf4
+  - numpy=1.24
```

Comment:
Not sure if this matters much in this case, but in general I prefer to use a `<1.25` indicator since this will make it less restrictive for the conda solver to find compatible packages.

Reply:
Updated to `numpy<1.25`